admin/acceptance.py

```diff
@@ -547,12 +547,19 @@ def main(reactor, args, base_path, top_level):
 
         control_node = nodes[0]
         dataset_backend = options.dataset_backend
+        # FLOC-1925 get the backend config and credentials from the
+        # supplied config.
```

Comment:
From `options["config-file"]`?  Do we have an example/other docs for the structure of that file?